front/src/components/group/index.js

```diff
@@ -1,31 +1,26 @@
 import React, {Component} from 'react';
-import {Typography, List, Avatar, Layout, notification, Result, Modal, message } from 'antd';
+import { Typography, List, Avatar, Layout, notification, Result, Modal, message } from 'antd';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
-import {EditOutlined, UserOutlined, ExclamationCircleOutlined} from '@ant-design/icons';
+import { UserOutlined, ExclamationCircleOutlined } from '@ant-design/icons';
 import Spinner from '../spinner';
-import CustomAvatar from '../avatar';
 import {
     StyledListItem, StyledTitle, StyledP, StyledDivBtn, StyledBtn,
-    StyledRow, GroupTitleP, DeleteStudentBtn, StyledListItemMeta, StyledStudentBlock,
-    StyledTeachersBlock, EditGroupBtn, StyledColHeader, StyledList
+    StyledRow, DeleteStudentBtn, StyledListItemMeta, StyledStudentBlock,
+    StyledTeachersBlock, StyledColHeader, StyledList
 } from './style';
-import GroupEditForm from '../groupEditForm';
-import { changeUpdatingStatus, deleteStudentFromGroup } from '../../common/redux/groups/groups.action';
-import { checkUserStatus } from '../../common/functions';
+import { deleteStudentFromGroup } from '../../common/redux/groups/groups.action';
+import GroupHeader from './groupHeader';
 
 const { Title } = Typography;
 const { Content } = Layout;
 const { confirm } = Modal;
 
 class Group extends Component {
-    constructor(props){
-        super(props);
-        this.state = {
-            role: checkUserStatus(),
-            fatalError: false,
-        };
-    }
+    state = {
```

Comment:
Please, remove this code, it is actually useless, in my opinion

Reply:
done